src/main/java/com/github/bbooong/bangumall/core/exception/BanguMallNullPointerException.java

```diff
@@ -0,0 +1,10 @@
+package com.github.bbooong.bangumall.core.exception;
+
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+
+public class BanguMallNullPointerException extends BanguMallException {
```

Comment:
```suggestion
public class BanguMallNotAllowedNullException extends BanguMallException {
```

NPE는 Null을 참조할 때도 발생하지만 이 예외는 null 값을 허용하지 않는 예외니까 `NotAllowedNull`도 괜찮을 것 같네요.